backend/mlarchive/tests/utils/decorators.py

```diff
@@ -29,3 +35,57 @@ def test_check_list_access_ok():
     func.assert_called_with(request)
     assert_equal(response, 'my response')
 '''
+
+
+def get_error_message(response):
+    content = response.content.decode('utf-8')
+    data_as_dict = json.loads(content)
+    return data_as_dict['error']
+
+
+def test_require_api_key(settings):
+    settings.API_KEYS = {'abcdefg': '/api/v1/message/'}
```

Comment:
Suggest using [`override_settings()`](https://docs.djangoproject.com/en/4.2/topics/testing/tools/#django.test.override_settings) to avoid the risk of contaminating other tests with your change here.

Reply:
This test uses the pytest-django settings fixture, https://pytest-django.readthedocs.io/en/latest/helpers.html#settings. Is override_settings() preferred?

Reply:
oh, didn't notice that. Never mind my knee-jerk reaction. :-)